rasa/shared/exceptions.py

```diff
@@ -77,3 +81,15 @@ class InvalidConfigException(ValueError, RasaException):
 
 class UnsupportedFeatureException(RasaCoreException):
     """Raised if a requested feature is not supported."""
+
+
+class InvalidEntityFormatException(RasaException):
+    """Raised if the format of an entity is invalid."""
+
+
+class ConnectionException(RasaException):
+    """Raised when a connection to a 3rd party service fails.
+
+    It's used by our broker and tracker store classes, when
+    they can't connect to services like postgres, s3, mongo.
```

Comment:
S3 is nor a broker nor a tracker store.

Reply:
I think I meant AWS. But i'll write DynamoDB